conda_build/environ.py

```diff
@@ -118,7 +145,9 @@ def get_dict(m=None, prefix=None):
             d['CPU_COUNT'] = "1"
 
     if m.get_value('source/git_url'):
-        d.update(**get_git_build_info(d['SRC_DIR']))
+        d.update(**get_git_build_info(d['SRC_DIR'],
+                                      m.get_value('source/git_url'),
+                                      m.get_value('source/git_rev', default='master')))
```

Comment:
Interesting, I saw the introduction of `git_rev` in the docs and was wondering what was going on. Will `git_tag` and `git_branch` then be deprecated at some point.